python/src/fragment.rs

```diff
@@ -388,6 +388,39 @@ pub fn write_fragments(
     Ok(export_vec(reader.py(), &fragments))
 }
 
+#[pyfunction(name = "_write_fragments_transaction")]
+#[pyo3(signature = (dest, reader, **kwargs))]
+pub fn write_fragments_transaction(
```

Comment:
Instead of creating a second method here that mirrors the first (difficult to maintain) can we add `return_transaction` to the `write_fragments` method?